app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -204,63 +176,6 @@ class PoliciesActivityTest {
       }
     }
   }
-  @Test
-  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
```

Comment:
We still want to test the intent part, just not the mock part (the mock is used for testing the fragment, and intents are used for testing the activity).

Reply:
Ah. I fixed it. Thanks.